apps/src/aichat/chatApi.ts

```diff
@@ -1,4 +1,9 @@
-import {Role, Status, ChatCompletionMessage, PII} from './types';
+import {
+  Role,
+  Status,
+  ChatCompletionMessage,
+  PII,
+} from '@cdo/apps/aiTutor/types';
```

Comment:
@sanchitmalhotra126 I'm not totally up to speed on how your team might be using or plan to use the chatApi so maybe let's connect on what we should share/split etc....and make sure we're not stepping on each other's toes!

Reply:
Sounds good! We'll likely reuse some of this for upcoming Gen AI work, but development hasn't started on that yet, so this is fine for now. We can likely pull these types into a common location if/when needed.